docs/UserGuide.md

```diff
@@ -37,9 +37,9 @@ of related information. Some tips and tricks, as well as cautions are included t
 help you get more out of the application and avoid certain common dangers. Lastly, please do use the table of contents
 at the top of the document for easier navigation throughout document. 
 
-Click here to return to the top:
+--------------------------------------------------------------------------------------------------------------------
+This marks the end of the section. Click here to return to the top:
```

Comment:
Can be a bit more specific with the "section", so use Introduction/Quick Start etc.

Reply:
noted and amend